website/source/docs/platform/k8s/helm/examples/ha-with-consul.html.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "docs"
+page_title: "Highly Available Vault Cluster with Consul"
+sidebar_current: "docs-platform-k8s-examples-ha-with-consul"
+sidebar_title: "Highly Available Vault Cluster with Consul"
+description: |-
+  Describes how to set up a highly available Vault cluster with Consul backend
+---
+
+# Highly Available Vault Cluster with Consul
+
+~> **Important Note:** This chart is not compatible with Helm 3. Please use Helm 2 with this chart.
+
+The below `values.yaml` can be used to set up a five server Vault cluster using
+Consul as a highly available storage backend, Google Cloud KMS for Auto Unseal.
+
+```yaml
+global:
+  enabled: true
+  image: "vault:1.3.0"
```

Comment:
Old version and this doesn't exist anymore. It's now under server. I suggest just removing the global stuff in general.

Reply:
:+1: Removed